### DISCUSSION TITLE
Auth token expiration?

### DISCUSSION THREAD

**Discussion by Tamooj** [cayossarian/span#7] [Q&A]
I used the integration to provision the auth, opening and closing the panel switch 3x. Everything seemed to work find, and I could see all the entities (sensors, switches, etc.).  Then after about ~30minutes the auth expired.  Wondering if the integration setup was just using the temporary unlocking from the panel, and not fetching/storing an auth token.  I can try to manually fetching a token and then reinserting it into the integration.  I guess my question is; did I get a token that simply expired (thanks, Span), or was I just riding along on the 15 minute one-off temp auth from the 3x switch?

**Comment by Tamooj** [comment]
BTW - Thanks Greg. This is a great integration!

**Comment by cayossarian** [comment]
@Tamooj I'm guessing you don't have the correct Span Integration installed as the current official HACs span integration has a bug and I think that repository is abandoned.  You need to install a **custom** repository that links to [https://github.com/SpanPanel/Span](url).  The readme at that link explains how to install a custom repository.  The version should be **v0.0.11** or greater.  We are trying to get this respository to be the default in HACs but there is a big HAC's catalog backlog.  Greg is the owner of the abandoned repository, this [repository](https://github.com/SpanPanel/Span) has an organization of maintainers and is the most current.   

cayossarian.
